user-integration-guide/symfony-integration.md

```diff
@@ -0,0 +1,80 @@
+# Integrating Opentelemetry PHP into Symfony Applications
+
+## Introduction 
+As a PHP developer I have always wondered how OpenTelemetry PHP could be beneficial to me. This user guide explains how one can instrument and gain insights from their Symfony application using the OpenTelemetry library. This guide highlights examples on using the OpenTelemtry PHP API. 
+## Set Up
+Ensure you have the following
+
+* This assumes that you have PHP Installed
+* You need [composer](https://getcomposer.org/download/) installed
+* You need [Symfony CLI](https://symfony.com/download) installed 
+* You need [docker](https://docs.docker.com/get-docker/) Installed
+
+## Step 1
+Create a Symfony application by running the command ` composer create-project symfony/skeleton my_project_name` for this example I am calling the project [otel-php-symfony-basic-example](https://github.com/prondubuisi/otel-php-symfony-basic-example)
+
+## Step 2
+Add [Symfony maker bundle](https://symfony.com/doc/current/bundles/SymfonyMakerBundle/index.html) to help us create boiler plate code easily by running `composer require symfony/maker-bundle --dev`. To easily define our routes within our controller methods lets require the Doctrine annotation library by running the command `composer require doctrine/annotations`
+
+Test that routes defined within Controllers work by creating a HelloController.php file within `src\Controller` folder
```

Comment:
`HelloControlller.php` can be inside of a code block, it might make it read more cleanly.